.github/component_owners.yml

```
@@ -17,67 +21,101 @@ components:
     - legendecas
   packages/opentelemetry-id-generator-aws-xray:
     - carolabadeer
+  packages/opentelemetry-propagation-utils: []
+    # Unmaintained?
   packages/opentelemetry-redis-common:
     - haddasbronfman
+  packages/opentelemetry-test-utils: []
+    # Unmaintained?
   plugins/node/instrumentation-amqplib:
     - blumamir
   plugins/node/instrumentation-dataloader:
     - henrinormak
   plugins/node/instrumentation-fs:
     - rauno56
+  plugins/node/instrumentation-lru-memoizer:
+    - blumamir
+  plugins/node/instrumentation-mongoose:
+    - blumamir
+  plugins/node/instrumentation-socket.io:
+    - mottibec
+    - rauno56
   plugins/node/instrumentation-tedious:
     - rauno56
   plugins/node/opentelemetry-instrumentation-aws-lambda:
     - carolabadeer
   plugins/node/opentelemetry-instrumentation-aws-sdk:
     - carolabadeer
     - blumamir
+  plugins/node/opentelemetry-instrumentation-bunyan: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-cassandra: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-connect: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-dns: []
+    # Unmaintained?
```

Originally contributed by @Flarna  

Would you care to take ownership?

```suggestion
  plugins/node/opentelemetry-instrumentation-dns:
    - Flarna
```

---

I haven't created and I don't want to take ownership.
I did only the conversion from plugin to instrumentation.
Can't remember who initially developed it. Best guess is that @OlivierAlbertini did a port from opencensus in the early days.